README.md

```diff
@@ -9,7 +9,7 @@
 </div>
 
 ## Quickstart 🏃‍
-First, make sure you have installed [Sketch](http://sketchapp.com) version 43 **(v44 is broken currently)**, & a recent [npm](https://nodejs.org/en/download/).
+First, make sure you have installed [Sketch](http://sketchapp.com) version 43 **(v44 is broken currently)**, & a recent [npm](https://nodejs.org/en/download/). - npm means Node.js (note for non-developers from a designer.) 
```

Comment:
The word "npm" here links directly to the node website; this additional sentence seems redundant to me.